ignite/contrib/handlers/clearml_logger.py

```diff
@@ -178,6 +178,24 @@ def bypass_mode(cls) -> bool:
         """
         return getattr(cls, "_bypass", bool(os.environ.get("CI")))
 
+    def __getattr__(self, attr: Any) -> Any:
+        """
+        Calls the corresponding method of ``ClearML.Logger``.
+
+        Args:
+            attr: methods of the ClearML.Logger class.
+        """
+        return getattr(self.clearml_logger, attr)
+
+    def get_task(self) -> Callable:
```

Comment:
Return type is incorrect here, let's make it `Any`

Reply:
Fixed.